backend/taco_backend.py

```diff
@@ -107,33 +125,37 @@ def solve_tri(A, B, lower=True, from_left=True, transp_L=False):
         else:
             return sla.solve_triangular(A, B, trans=transp_L, lower=lower)
 
+    @staticmethod
+    def array_equal(A, B):
```

Comment:
In general, are these data transformations (pt.from_array, to_array) makes taco slow?

Reply:
I will start some profiling as the next step.

Reply:
Actually, did some profiling, yes.

My guess is taco unwrap the whole numpy array into a 

for ele in np.array:
  taco_tensor.insert(ele)

Reply:
OK, sounds good